src/inference/enumerate.ad.js

```diff
@@ -47,6 +57,17 @@ module.exports = function(env) {
     return this.wpplFn(_.clone(this.store), env.exit, this.a);
   };
 
+  // Error function for error handling
+  // this.throwOnError is true: directly throw error
+  // this.throwOnError is false: return error (string) as infer result
+  Enumerate.prototype.error = function(errType) {
```

Comment:
The `err` is going to be printed to console in header.wppl. I guess we don't want see a few info messages starting with "Error" when doing default infer?